Oracle/PCORNetLoader_ora.sql

```diff
@@ -1748,27 +1748,57 @@ PMN_EXECUATESQL(sqltext);
 
 insert into dispensing (
 	PATID
-    ,PRESCRIBINGID
+  ,PRESCRIBINGID
 	,DISPENSE_DATE -- using start_date from i2b2
-    ,NDC --using pcornet_med pcori_ndc - new column!
-    ,DISPENSE_SUP ---- modifier nval_num
-    ,DISPENSE_AMT  -- modifier nval_num
+  ,NDC --using pcornet_med pcori_ndc - new column!
+  ,DISPENSE_SUP ---- modifier nval_num
+  ,DISPENSE_AMT  -- modifier nval_num
 --    ,RAW_NDC
 )
 /* Below is the Cycle 2 fix for populating the DISPENSING table  */
-select distinct
-  ibf.patient_num patid,
+with disp_status as (
+  select ibf.patient_num, ibf.encounter_num, ibf.concept_cd, ibf.instance_num, ibf.start_date, ibf.modifier_cd
+  from i2b2fact ibf
+  join BLUEHERONMETADATA.pcornet_med pnm

```

Comment:
Good catch on forgetting to swap the `"&&i2b2_meta_schema"` schema reference back in.